src/schema/common.ts

```diff
@@ -148,6 +150,27 @@ export const getSearchQuery = (param: string): string =>
 export const processSearchQuery = (query: string): string =>
   query.trim().split(' ').join(' & ') + ':*';
 
+export const meiliOffsetGenerator = <
+  TReturn extends { id: string },
+>(): PageGenerator<
+  TReturn,
+  ConnectionArguments & { ids: string[]; pagination: MeiliPagination },
+  OffsetPage,
+  unknown
+> => ({
+  connArgsToPage: ({ pagination }): OffsetPage => {
+    return {
+      limit: pagination?.total || 10,
+      offset: pagination?.offset || 0,
+      current: pagination?.current,
+    };
+  },
+  nodeToCursor: (page, { ids }, node): string =>
```

Comment:
ids, is just the current ones, we need to determine which one is actually there (exclude deleted ones)